src/csharp/Microsoft.Spark.E2ETest/IpcTests/Sql/FunctionsTests.cs

```diff
@@ -568,7 +571,7 @@ public void TestSignaturesV2_3_X()
             Assert.IsType<Column>(JsonTuple(col, "a"));
             Assert.IsType<Column>(JsonTuple(col, "a", "b"));
 
-            var options = new Dictionary<string, string>() { { "hello", "world" } };
+            var options = new Dictionary<string, string>() {{"hello", "world"}};
```

Comment:
do you know why this formatting changed? I think the existing formatting is fine. (Ctrl + K, D gives me the existing formatting). Could you double check the latest commit?

Reply:
I think resharper got a bit excited :)